### DISCUSSION TITLE
tdms_index

### DISCUSSION THREAD

**Discussion by vcmorini** [adamreeve/npTDMS#270] [General]
Often when tdms files are saved, alongside it is also saved a tdms_index file.
I wonder if npTDMS offers this solution, or how can it be implemented.

**Comment by adamreeve** [comment]
Hi @vcmorini, npTDMS can make use of the tdms_index files when reading, but doesn't currently create them. I don't think it would be too hard to implement this, as the tdms_index files just contain all of the metadata from the tdms file but without the data. I'd be happy to accept a pull request for this if it was optional, eg. adding a `create_index` option when reading a file.

**Comment by Eifi1** [comment]
Without seeing this discussion beforehand I created a Pull Requests that solves exactly that issue.

**Reply by adamreeve** [reply]
This is the PR link for reference: https://github.com/adamreeve/npTDMS/pull/278. This will be included in the next 1.6.0 release.

**Reply by Eifi1** [reply]
When do you think will the release 1.6.0 happen? Can I support in any way? What has to be made for a new release?

**Reply by adamreeve** [reply]
Sorry for the slow reply, I had some other work I wanted to get into 1.6.0 but have been too busy with other things, so I've released 1.6.0 with just this change now (plus another change to the supported versions that was already on master).
